### PR TITLE
Sorting wishlist and added docstrings

### PR DESCRIPTION
**Pull request by AlexChenl3**
fixed reviewStar and reviewCount sorts so now all the sorts are completed. docstrings are also added

### CODE REVIEW

**Review by PyroPoro — DISMISSED** [review]

File: src/main/java/ItemDateComparator.java [hunk 1]
```diff
@@ -2,6 +2,12 @@
...
     //compares two items by their dates, where the more recent the date, the higher the order
+    /**
+     * Takes 2 Items and compares their added dates using .getItemDateAdded() and sorts in ascending order (earliest
+     * date-latest date)
+     * @param item1 The first name of Item
+     * @param item2 The second name of Item
+     */
     @Override
     public int compare(Item item1, Item item2){
         if(item1.getItemDateAdded().after(item2.getItemDateAdded())){
```
Comment (Ljime) [inline src/main/java/ItemDateComparator.java:13]: Doc string doesn't match what the function does (Date sorter)

File: src/main/java/ItemNameComparator.java [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+    /**
+     * Takes 2 Items and compares their names using .getItemName() and sorts in ascending order (A-Z)
+     * @param item1 The first name of Item
+     * @param item2 The second name of Item
+     */
+    @Override
+    public int compare(Item item1, Item item2) {
+//        if (item1.getItemName().equals(item2.getItemName())) {
+//            return 1;
+//        }
```
Comment (Ljime) [inline src/main/java/ItemNameComparator.java:13]: Unused code here

File: src/main/java/ItemPriceComparator.java [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+public class ItemPriceComparator implements Comparator<Item>{
+    /**
+     * Takes 2 Items and compares their prices using .getItemPrice() and sorts in ascending order (lowest-highest)
+     * @param item1 The first name of Item
+     * @param item2 The second name of Item
+     */
+
+    @Override
+    public int compare(Item item1, Item item2) {
+        if (item1.getItemPrice() >= item2.getItemPrice()) {
```
Comment (Ljime) [inline src/main/java/ItemPriceComparator.java:11]: Doc string doesn't match what the function does (Price sorter)

File: src/main/java/ItemReviewCountComparator.java [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+import java.util.Comparator;
+
+public class ItemReviewCountComparator implements Comparator<Item> {
+    /**
+     * Takes 2 Items and compares their review count using .getReviewCount() and sorts in descending order (highest-lowest)
+     * @param item1 The first name of Item
+     * @param item2 The second name of Item
+     */
+    @Override
+    public int compare(Item item1, Item item2) {
```
Comment (Ljime) [inline src/main/java/ItemReviewCountComparator.java:10]: Doc string doesn't match what the function does

File: src/main/java/ItemReviewStarComparator.java [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+    // compares two items by their review star
+    // L-H as default
+    /**
+     * Takes 2 Items and compares their review stars using .getReviewStar() and sorts in descending order (highest-lowest)
+     *
+     * @param item1 The first name of Item
+     * @param item2 The second name of Item
+     */
+    @Override
+    public int compare(Item item1, Item item2) {
```
Comment (Ljime) [inline src/main/java/ItemReviewStarComparator.java:12]: Docstring doesn't match what the function does (Review stars)

**Comment by AlexChenl3** [comment]
I updated the docstrings and deleted commented code, please check them again to see if there are any more issues. If not i will merge into main

**Comment by Ljime** [comment]
> I updated the docstrings and deleted commented code, please check them again to see if there are any more issues. If not i will merge into main

Thanks for the changes LGTM!

**Review by Ljime — APPROVED** [review]

**Review by PyroPoro — APPROVED** [review]

**Review by DavidZ79 — APPROVED** [review]

**Pull request merged**